src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Button from "./Button";
+import "../ButtonPanel.css";
+export default function ButtonPanel() {
+  const btnGroup1 = ["AC", "+/-", "%", "÷"];
+  const btnGroup2 = ["7", "8", "9", "X"];
+  const btnGroup3 = ["4", "5", "6", "-"];
+  const btnGroup4 = ["1", "2", "3", "+"];
+  const btnGroup5 = ["0", ".", "="];
+  return (
+    <div>
+      <div class="buttons">
```

Comment:
Let us fix this error

![image](https://user-images.githubusercontent.com/6015315/64136817-f3732700-ce26-11e9-83ca-f4781787407a.png)